### DISCUSSION TITLE
How do the cache and store credential storage methods differ?

### DISCUSSION THREAD

**Discussion by beedell-roke** [Murmele/Gittyup#521] [Q&A] [answered]
I want to store all credentials permanently.

1.	> ![image](https://user-images.githubusercontent.com/42837531/232847280-049b77b9-149a-4701-94ea-31c3c016528f.png)
1.	> ![image](https://user-images.githubusercontent.com/42837531/232847332-6727f57a-3210-41a6-baef-c56fb6d11593.png)

How do cache and store differ?

**Comment by gh-devnull** [comment, marked as answer]
"cache" stores the password temporarily (defaults to 15 minutes) in memory.  "store" stores the password on disk.  For more detail, check out the documentation for git-credential-cache and git-credential-store.
